onnxscript/optimizer/_inliner.py

```diff
@@ -142,10 +147,11 @@ def clone_graph(self, graph: ir.Graph) -> ir.Graph:
         input_values = [self.clone_value(v) for v in graph.inputs]
         nodes = [self.clone_node(node) for node in graph]
         initializers = [self.clone_value(init) for init in graph.initializers.values()]
+        output_values = [self.clone_value(v) for v in graph.outputs]
```

Comment:
So this will simply retrieve the values from the value pool (instead of actual cloning). Is that the correct understanding? If so a comment will be helpful I think.

Reply:
Yes. Real cloning happens at places which introduce a symbol (which are graph inputs, and outputs of nodes). At places which just reference (consume) a symbol, we just retrieve the value.